modules/serverless-quarkus-function-return-values.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies
+//
+// * /serverless/functions/serverless-developing-quarkus-functions.adoc
+
+[id="serverless-quarkus-function-return-values_{context}"]
+= Quarkus function return values
+
+Functions can return an instance of:
+
+* Any type from the list of permitted types.
+* The `Uni<T>` type, where the `<T>` type parameter can be of any type from the permitted types.
+
+The `Uni<T>` type is useful if a function calls asynchronous APIs, because the returned object is serialized in the same format as the received object. For example:
+
+* If a function receives an HTTP request, then the returned object is sent in the body of an HTTP response.
+* If a function receives a `CloudEvent` object in binary encoding, then the returned object is sent in the data property of a binary-encoded `CloudEvent` object.
+
+.Example command
+[source,java]
+----
+public class Functions {
+    @Funq
+    public List<Purchase> getPurchasesByName(String name);
```

Comment:
I know it is just example but that will not compile.. (due the ';' ). maybe change to...

```
public List<Purchase> getPurchasesByName(String name) {
  // logic to retrieve purchases
}
```